src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,16 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.Password;
+
+@Password
 public class UserRequestDto {
+    @Email
     private String email;
+    @Min(8)
     private String password;
+    private String repeatPassword;
```

Comment:
Add size constraints for all fields.
Also, a good practice to limit max length since our DB column has a limited length and can cause `SQLException` so better to avoid it